abstract/group.go

```diff
@@ -51,6 +51,9 @@ type Secret interface {
 
 	// Set to a fresh random or pseudo-random secret
 	Pick(rand cipher.Stream) Secret
+
+	// SetBytes will take bytes and create a secret out of it
+	SetBytes([]byte) Secret
```

Comment:
1) There's no immediate reason to put it in this PR since it won't be used (But still can!)
2) Do you have the exact functionality of this function in mind ? 
 * `Marshal/Unmarshal` will give/read the representation of a secret correctly formatted and everything
 * `SetBytes` is supposed to accept any bytes input and squeeze that into a scalar. So I guess `Bytes()` is supposed to give back the original slice (1) ? Or does it returns the same thing as `MarshalBinary` (2) ? 
IF (1), then we need to keep track of an original slice alongside with the scalar (nist.Int).
IF (2), then there's absolutely no difference between `MarshalBinary` and `Bytes` 

I'm not so sure about both options, so what do you have in mind for the `Bytes()` function...